src/main.ts

```diff
@@ -29,11 +31,13 @@ export default function main() {
   createXStateV4Machine(generatorOptions)
   // --------------------------------------------------
 
-  console.log(writer.toString())
+  const content = writer.toString()
+  const options = { width: 480, height: 240 }
+  showUI(options, { content })
```

Comment:
**Thought**: I love the UI and I just realized the guide does not touch the topic of auto-reload in Figma!